### PR TITLE
Update sonarr.markdown

### PR DESCRIPTION
**Pull request by jwildman16**
Added information on how to configure the integration. I don't understand how to properly format the text, so additional changes may be needed.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
This will not render as you think it does:

image

Could you please adjust? Please note, we are using Markdown here not normal text files.

**Comment by jwildman16** [comment]
I'm up for it, but I'd need a little guidance. It's outside of my current skill set. If you have a self-guided learning resource to recommend, I'm happy to check it out and revise my entry. Thanks.

File: source/_integrations/sonarr.markdown [hunk 1]
```diff
@@ -16,4 +16,21 @@ The `Sonarr` integration pulls data from a given [Sonarr](https://sonarr.tv/) in
 
 ## Configuration
 
-Go to the integrations page in your configuration and click on new integration -> Sonarr.
+Go to the Configuration > Integrations page and click on the plus '+' to add a new integration. Select Sonarr. Enter your Sonarr connection information and click Submit.
```
Comment (zsarnett) [inline source/_integrations/sonarr.markdown:19]: We can now use a new plugin called `configuration_basic` 

Checkout the documentation for the Lovelace Weather card on how to use this plugin

Something along the lines of this
```markdown
{% configuration_basic %}

(Here you will place your variables

{% endconfiguration_basic %}
```

**Comment by frenck** [comment]
This PR is running stale, closing it for now. Feel free to re-open when ready to work on it again.

**Pull request closed without merging**